src/UglyToad.PdfPig/Writer/TokenWriter.cs

```diff
@@ -219,12 +219,11 @@ public void WriteToken(IToken token, Stream outputStream)
                      * n is a literal keyword identifying this as an in-use entry
                      * eol is a 2-character end-of-line sequence ('\r\n' or ' \n')
                      */
-                        var paddedOffset = OtherEncodings.StringAsLatin1Bytes(offset.Offset.ToString("D10", CultureInfo.InvariantCulture));
+                        var paddedOffset = Encoding.ASCII.GetBytes(offset.Offset.ToString("D10", CultureInfo.InvariantCulture));

```

Comment:
why changing that?

Reply:
Is a bit faster to use the ASCII encoder on ASCII only digits (and we're guaranteed to only get back ASCII digits here). I had planned to optimize this on #NET later to encode directly to an ASCII byte[] buffer.